server/lib/models/import.go

```diff
@@ -0,0 +1,32 @@
+package models
+
+import (
+	"log"
+	"path/filepath"
+	"os"
+)
+
+func create_channels(path string) {
```

Comment:
__[gometalinter]__
* [golint] don't use underscores in Go names; func create_channels should be createChannels